cmd/plan/pl006/plan.go

```diff
@@ -7,36 +7,32 @@ import (
 )
 
 // Plan describes in which order and with which tolerance to execute actions of
-// this cluster scope.
+// this test plan.
 var Plan = []plan.Step{
 	{
-		Action:  "ac001",
+		Action:  "create/cluster/singlecontrolplane",
 		Backoff: plan.NewBackoff(10*time.Second, 2*time.Second),
-		Comment: "create cluster CRs",
 	},
 	{
-		Action:  "ac002",
+		Action:  "verify/cluster/created",
 		Backoff: plan.NewBackoff(30*time.Minute, 3*time.Minute),
-		Comment: "check cluster access",
 	},
 	{
-		Action:  "ac003",
+		Action:  "update/cluster/ha",
```

Comment:
Below I see `verify/cluster/hasetup`. Can we keep the action names across semantics aligned? So that we either go for `ha` or `hasetup` in `update` and `verify`. Or is there a deeper logic to it?

Reply:
renamed to ha, better to have create/verify with same name